emeritus-tech/landing-pages-libraries
# [CPT-520] Berkeley stylesheet and markup: DT-AI-DT

🎟 [Jira Ticket CPT-520](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-520)
🎟 [Jira Ticket CPT-521](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-521)
🎟 [Jira Ticket CPT-522](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-522)

🚢 Deployment Instructions

N/A

✅ QA instructions

** Testimonial slider should work as expected using CSS only (not entirely the same as the original landing page one)

**To test Data Science (online):**

Run command npm start
Open http://localhost:8080/BDT.html
And verify on browser:
Design should look exactly as original states [landing CP](https://em-executive.berkeley.edu/data-science?ab_test[e20191209_static_landing_pages_course_preview]=variant), [landing legacy](https://em-executive.berkeley.edu/data-science?ab_test[e20191209_static_landing_pages_course_preview]=control), [thank you]( https://em-executive.berkeley.edu/data-science/thank-you) and [apply](https://emeritus-executive.berkeley.edu/data-science/apply.php).

🖼 Previews

**Data Science (Online):**

![screencapture-localhost-8081-BDS-html-2020-05-25-09_23_58](https://user-images.githubusercontent.com/58177278/82812963-86973580-9e6a-11ea-8d4e-0e68667d3c51.png)
